gopro_webcam.service

```diff
@@ -4,7 +4,7 @@ After=network-online.target
 Wants=network-online.target systemd-networkd-wait-online.service
 
 [Service]
-ExecStart=/usr/local/sbin/gopro webcam -p enx -a -n
+ExecStart=/usr/bin/gopro webcam -a -n
```

Comment:
you start `/usr/bin/gopro` but copy the gopro script in `/usr/local/sbin/gopro`

Reply:
Yeah, that was a change for the AUR actually. I think would be best if we keep it in `/usr/local/sbin/`. Let me fix that.